src/controller.jl

```diff
@@ -72,9 +72,10 @@ function compute_controller_reach!(contr, autom, initlist, targetlist)
     # nexttargetlist = Int[]
     soursymblist = Tuple{Int,Int}[]
 
-    prog = ProgressUnknown("# iterations computing controller:")
+    # Commented because it changes the number of allocations
```

Comment:
You mean
```
if verbose
    prog = ProgressUnknown("# iterations computing controller:")
end
```
?
Ok for `compute_symmodel_from_controlsystem` but for `compute_controller_reach/safe` would the addition of a branch instruction in the main loop not decrease the performances? :)

Reply:
It should be completely negligeable compared to the rest of the computation of the loop.

Reply:
I added it for `compute_controller_reach` but it doesn't work as expected: all the information is printed at the end... Also I did not know how to add it for `compute_symmodel_from_controlsystem` since it requires a for loop as second argument ...

Reply:
Ok, leave it like this, i'll take a look